pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -115,8 +115,12 @@ func (r *REST) Update(ctx kapi.Context, tagName string, objInfo rest.UpdatedObje
 			return nil, false, kapierrors.NewBadRequest("namespace is required on ImageStreamTags")
 		}
 		imageStream = &api.ImageStream{
-			ObjectMeta: kapi.ObjectMeta{Namespace: namespace, Name: name},
+			ObjectMeta: kapi.ObjectMeta{
+				Namespace: namespace,
+				Name:      name,
+			},
 		}
+		kapi.FillObjectMetaSystemFields(ctx, &imageStream.ObjectMeta)
```

Comment:
Updates now require objects have valid UIDs.  Agree that we need something
cleaner, but in this case the tag actually does have a UID for
preconditions (the underlying image UID).  Dunno how I feel about lying
about the object UID.

On Wed, Jul 6, 2016 at 10:00 AM, David Eads <notifications@github.com>
wrote:

> In pkg/image/registry/imagestreamtag/rest.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69735076>:
>
> >  		}
> > +		kapi.FillObjectMetaSystemFields(ctx, &imageStream.ObjectMeta)
>
> This is being done to fake the new UpdatedObject call that @liggitt
> <https://github.com/liggitt> added? There's got to be a better way to
> handle create on update calls.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/993c4ac1fd086af40ae0e0c811ce9052ce185714..be6ceebe60f75425e3317b336ed15b7354e9f4b1#r69735076>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p5LGvkqp1C8LRPoTI2WoVYYaoExcks5qS7UDgaJpZM4I-xTE>
> .
>